data/transactions/logic/eval.go

```diff
@@ -2062,6 +2062,29 @@ func opTxna(cx *evalContext) {
 	cx.stack = append(cx.stack, sv)
 }
 
+func opTxnas(cx *evalContext) {
+	last := len(cx.stack) - 1
+
+	field := TxnField(uint64(cx.program[cx.pc+1]))
```

Comment:
Let's remove the inner cast, which doesn't do anything.  (I know we're doing this all over the place, I have a PR I'm working on the fix the existing ones.)

Reply:
Good point - I'll delete the new inner casts